src/main/java/io/strimzi/kafka/bridge/SinkBridgeEndpoint.java

```diff
@@ -556,4 +564,14 @@ protected void consume(Handler<KafkaConsumerRecords<K, V>> manualRecordBatchHand
             }
         });
     }
+
+    protected void commit(Map<TopicPartition, io.vertx.kafka.client.consumer.OffsetAndMetadata> offsetsData, Handler<Map<TopicPartition, io.vertx.kafka.client.consumer.OffsetAndMetadata>> commitOffsetsHandler){
+        this.commitOffsetsHandler = commitOffsetsHandler;
+
+        this.consumer.commit(offsetsData, result -> {
+            if (result.succeeded() && this.commitOffsetsHandler != null) {
```

Comment:
You should handle the case when commit fails so result.succeded() is false. Maybe instead of a Handler<Map....> you should use Handler<AsyncResult<Map...>> so that the AsyncResult can be set to be failed.